python/lsst/ts/MTAOS/MtaosCsc.py

```diff
@@ -124,15 +188,33 @@ def getModel(self):
         corrections to the AOS (active optical system) subsystems.
         """
 
-        timestamp = self._getTimestamp()
-        self.model.resetWavefrontCorrection()
+        self._logExecFunc()
+        self._checkEnabledState()
```

Comment:
I added that layer (`_checkEnableState()`) because I want to put the error message to the log file. I can check the function of `self.assert_enabled` in the ts_salobj 3.5 (after the release) in the next update of `ts_MTAOS`.

Reply:
Just to be clear: `assert_enabled(action)` has been supported by ts_salobj for a long time. new in v3.5 is the ability to omit the `action` argument.

Reply:
Will check the update of `ts_sal` 3.5 when the related docker image is ready. Thanks!